### PR TITLE
Update Account UI French translation as of September 26th.

### PR DESCRIPTION
**Pull request by DimitriDR**
Hi,

This is my contribution to add the few new strings for Account UI.
Also fixed some little typos previously written.

### CODE REVIEW

**Comment by jonkoops** [comment]
@Paul-Van-Uytvinck since you previously worked on the French localizations could I ask you to review this PR?

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 1]
```diff
@@ -157,5 +157,18 @@
   "updateSuccess": "Ressource mise à jour avec succès.",
   "user": "Utilisateur",
   "username": "Nom d'utilisateur",
-  "usernamePlaceholder": "Nom d'utiisateur ou courriel"
+  "usernamePlaceholder": "Nom d'utilisateur ou courriel",
+  "groupsListHeader": "En-tête de la liste des groupes",
+  "groupsListColumnsNames": "Nom des colonnes de la liste des groupes",
+  "aliasHelp": "Nom de la configuration",
+  "flowTypeHelp": "Quel genre de formulaire est-ce",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:164]:
```suggestion
  "flowTypeHelp": "De quel genre de formulaire s'agit-il",
```

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 2]
```diff
@@ -157,5 +157,18 @@
...
+  "createFlowHelp": "Vous pouvez créer un flow de premier niveau au sein de ce formulaire",
+  "rolesScope": "S'il n'y a aucun mappage de portée de rôle défini, chaque utilisateur est autorisé à utiliser cette portée client. S'il y a des mappages de portée de rôle, l'utilisateur doit être un membre d'au moins d'un des rôles.",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:166]:
```suggestion
  "rolesScope": "S'il n'y a aucun mappage de portée de rôle défini, chaque utilisateur est autorisé à utiliser cette portée du client. S'il y a des mappages de portée de rôle, l'utilisateur doit être membre d'au moins l'un des rôles.",
```

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 3]
```diff
@@ -157,5 +157,18 @@
...
+  "scopeNameHelp": "Nom de la portée du client. Doit être unique dans le domaine. Le nom ne doit pas contenir d'espaces car il est utilisé comme valeur du paramètre de la portée",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:167]:
```suggestion
  "scopeNameHelp": "Nom de la portée du client. Doit être unique dans le domaine. Le nom ne doit pas contenir d'espace car il est utilisé comme valeur du paramètre de la portée",
```

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 4]
```diff
@@ -157,5 +157,18 @@
...
+  "scopeDescriptionHelp": "Description de la portée du client",
+  "scopeTypeHelp": "Portés du client, qui seront ajoutées comme portées par défaut à chacun des clients créés",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:169]:
```suggestion
  "scopeTypeHelp": "Portées du client, qui seront ajoutées comme portées par défaut à chacun des clients créés",
```

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 5]
```diff
@@ -157,5 +157,18 @@
...
+  "clientDescriptionHelp": "Spécifie la description du client. Par exemple, « Mon client pour TimeSheet ». Prend également en charge les clés pour les valeurs localisées. Par exemple : ${my_client_description}",
+  "clientTypeHelp": "Le type de cette ressource. Il peut être utilisé pour grouper ensemble différentes instances de ressources avec le même type.",
+  "scopesHelp": "Les portées associées avec cette ressource."
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:172]:
```suggestion
  "scopesHelp": "Les portées associées à cette ressource."
```

**Review by Paul-Van-Uytvinck — CHANGES_REQUESTED** [review]

**Review by Paul-Van-Uytvinck — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
Thanks again @DimitriDR @Paul-Van-Uytvinck!

**Pull request merged**